docs/setup/configuration.mdx

```diff
@@ -152,12 +152,6 @@ Some examples:
 
 **Example:** `smtp://user:password@smtp.example.com:465/?ssl=True`
 
-### `ENABLE_ACCOUNT_CONFIRMATION_BY_EMAIL`
```

Comment:
Is it expected that we removed it? 🤔

Reply:
Yes, I included not only changes from pr: https://github.com/saleor/saleor/pull/13059
But also from this one: https://github.com/saleor/saleor/pull/12781